src/_content/posts/call-for-2025-volunteers.md

```diff
@@ -0,0 +1,319 @@
+
+- Determine what roles need on-site volunteer support (session chairs, session managers, registration desk, etc.)
+- Set up sign-up for on-site volunteers and online volunteers (moderators/cheerleaders)
+- Draft announcements for calls for volunteers
+
+### At-conference responsibilities
+
+- Tell volunteers where to go
+- Give volunteers tools to succeed
+
+## Communications Committee
+
+Size: 2-4
+
+The communications committee is responsible for getting the word out about DCUS.
+
+### Pre-conference responsibilities
+
+- Publish announcements and reminders on social media, the conference blog, and more (Django News, Python Community News, etc.)
+- Send copy to the Django Software Foundation to get major news events on the Django blog
+- Review, edit, and suggest copy for the other teams' announcements
+- Tweet and toot. Lots of tweeting and tooting.
+- Select an "official emoji" for the conference (typically suggested by the design package for the year's website)
+- Boost and retweet speakers and organizers talking about the conference
+- Schedule tweets and toots announcing talks and events
+
+### At-conference responsibilities
+
+- Pay attention to the `#DjangoCon` and `#DjangoConUS` topics on social media and mentions to `@djangocon` on Twitter
+- Boost speakers and organizers
+- Answer questions as they come up
+- Run the `#help-desk` channel on the conference attendee slack
+
+## Website Committee
+
+Size: 2-3
+
+The website committee is responsible for updating the website as the conference transitions through phases (CFP coming soon, CFP open, CFP closed, schedule released, hotel block closed, conference over, etc.), reviewing pull requests, and adding content (e.g. new sponsors). This requires the most coding of any team aside from maybe the program team.
+
+### Pre-conference responsibilities
+
+- Update static pages for CFP status
+- Update static pages for venue information
+- Review pull requests from other teams
+- Add sponsors as directed by sponsorship team
+
+## Automation Team
+
+This team is more free-flowing and writes scripts to make teams' lives easier. Responsibilities vary based on the year, an if you have ideas for special projects for the future, have at it!
+
+## Sprints Team
+
+Size: 2-4
+
+### Pre-conference responsibilities
+
+- Write blog posts announcing sprints
+- Invite people to launch special sessions (e.g. Carlton's ["Getting Started Contributing to Django"](https://2022.djangocon.us/sprints/getting-started-contributing-to-django/))
+- Ensure we have signage and power equipment
+
+### At-conference responsibilities
+
+- Coordinate help desk
+- Direct people to lunches and breaks
+
+## Wrap-up
+
+To sign up for any of the teams (or just say "put me to work"), email hello@djangocon.us, and we'll put you on the list for the 2025 kickoff meeting. We would love to have you! You do not have to have previous organizing experience (although we might prefer it for some leadership roles).
```

Comment:
Are we removing the "Health & Safety" committee next year?

Reply:
I didn't think to include it in my 2023 version and it didn't get added here. Want to put something together for it? (and/or @jcjudkins)

Reply:
```
## Health & Safety

Size: 1-2

The health and safety committee enacts the health and safety policy each year. The policy itself is decided by DEFNA and is not part of this committee's responsibilities. 

### Pre-conference responsibilities

- Order COVID tests and masks (if needed per policy)
- Ensure that the health & safety check-in has volunteers
- Coordinate the health and safety blog posts with the Comms team

### At-conference responsibilities

- Spend some volunteer time at the health & safety check-in (if it is part of the policy that year)
- Answer health & safety questions as needed

```